src/cryptoadvance/specter/controller.py

```diff
@@ -141,6 +141,11 @@ def settings():
     protocol = 'http'
     explorer = app.specter.explorer
     auth = app.specter.config["auth"]
+    version_info={}
+    version_info["current"], version_info["latest"], version_info["upgrade"] = get_version_info()
+    app.logger.info("Upgrade? {}".format(version_info["upgrade"]))
+    if version_info["upgrade"]:
+        flash("There is a new version available. Consider strongly to upgrade to the new version {}".format(version_info["latest"]))
```

Comment:
Maybe makes sense to add a hint how to update, like:
`Update to {version_info["latest"]} with "pip3 install {name} --upgrade"`
and get the package `name` from `get_version_info()` as well

Reply:
Not sure why the package_name should get returned. It's cryptoadvance.specter, right?
Fiex the other stuff.